tests/multinode.rs

```diff
@@ -668,6 +668,7 @@ fn retry_get_balance(
                 return Some(o);
             }
         }
+        sleep(Duration::from_millis(20));
```

Comment:
I wonder, is this loop around `poll_get_balance` dated? That function already sleeps between polling. https://github.com/solana-labs/solana/blob/3beb38ac8abd0c6de562b870b20aba2a0415ec4b/src/thin_client.rs#L210

Reply:
Maybe it's not needed. I'll try without it..